composer.json

```diff
@@ -28,12 +29,33 @@
 	},
 	"config": {
 		"allow-plugins": {
-			"dealerdirect/phpcodesniffer-composer-installer": true
+			"dealerdirect/phpcodesniffer-composer-installer": true,
+			"civicrm/composer-downloads-plugin": true
 		}
 	},
 	"authors": [
 		{
 			"name": "Elementor team"
 		}
-	]
+	],
+	"extra": {
+		"downloads": {
```

Comment:
It's not clear to me when civicrm/composer-downloads-plugin will decide to download a new version to override the previous version downloaded.